ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -60,22 +46,9 @@ tests:
   cron: 6 16 2 * *
   steps:
     cluster_profile: azure-qe
+    env:
+      ARO_CLUSTER_VERSION: 4.13.23
```

Comment:
> intent here is to install 4.13.23 then upgrade to latest nightly, same as we do for 4.12 and 4.14

If that, suggest to update the yaml file name, something like `openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro-4.13.yaml`, also `zz_generated_metadata.variant` in the yaml file, so that help us quickly get know what upgrade path is being covered.

Reply:
renamed files and updated metadata.variant.